MergerLogic/DataTypes/Data.cs

```diff
@@ -267,8 +265,7 @@ public bool TileExists(Coord coord)
 
         public Tile? GetCorrespondingTile(Coord coords, bool upscale)
         {
-            Stopwatch fetchTileStopWatch = new Stopwatch();
-            fetchTileStopWatch.Start();
+            Stopwatch fetchTileStopWatch = Stopwatch.StartNew();
```

Comment:
This variable only has this function as it's scope.
Please rename to something like `stopwatch`.